packages/context/src/oidcContext/AuthenticationContext.provider.tsx

```diff
@@ -165,7 +165,22 @@ export const AuthenticationProviderInt = ({
   );
 };
 
-const AuthenticationProvider: FC = withRouter(
+type AuthenticationProviderProps = Pick<AuthenticationProviderIntProps,
+  | 'location'
+  | 'history'
+  | 'configuration'
+  | 'isEnabled'
+  | 'UserStore'
+  | 'loggerLevel'
+  | 'logger'
+  | 'sessionLostComponent'
+  | 'authenticating'
+  | 'notAuthenticated'
+  | 'notAuthorized'
+  | 'callbackComponentOverride'
+  >
```

Comment:
Why choose Pick utility instead of Omit ? It could be simpler isn't it ? (real question, I dont know :))

Reply:
Real answer to a real question ;-) i tested both but i didn't know how to choose the "right" one. This one seems easier to maintain (IMO). But i can change :-)

Picks seems to be more explicit about the "public" properties. What do you think?

Reply:
My reasoning => If you use omit instead of Pick, you can add some props without refactoring this. You just had to omit all injected props.

@guillaume-chervet What do you think about it ?

Reply:
To move on (Guigui is quite busy) can you change and use omit. I discuss about it with pairs and it seems to be the best way.

Reply:
Done